lib/screens/add_recipe_screen.dart

```diff
@@ -30,41 +53,64 @@ class _AddRecipeScreenState extends State<AddRecipeScreen> {
           child: Column(
             children: <Widget>[
               TextFormField(
+                style: const TextStyle(
+                  color: Colors.grey,
+                  fontFamily: 'EPAM2',
+                  fontWeight: FontWeight.w500,
+                  fontSize: 20,
+                ),
                 controller: _nameController,
                 decoration: const InputDecoration(labelText: 'Recipe Name'),
                 validator: (value) {
-                  if (value == null || value.isEmpty) {
-                    return 'Please enter a recipe name';
-                  }
-                  return null;
+                  return _validate(value ?? '', 'Please enter a recipe name');
                 },
               ),
               TextFormField(
+                style: const TextStyle(
+                  color: Colors.grey,
+                  fontFamily: 'EPAM2',
+                  fontWeight: FontWeight.w500,
+                  fontSize: 20,
+                ),
                 controller: _imageUrlController,
                 decoration: const InputDecoration(labelText: 'Image URL'),
                 validator: (value) {
-                  if (value == null || value.isEmpty) {
-                    return 'Please enter an image URL';
-                  }
-                  return null;
+                  return _validateUrl(value ?? '', 'Please enter a valid URL');
```

Comment:
instead ```dart 
validator: (value) {
return _validateUrl(value ?? '', 'Please enter a valid URL')
}
```
you can use 
```dart 
validator: (value) => _validateUrl(value ?? '', 'Please enter a valid URL')
```

Reply:
It was giving error that way, let me check if I can do it